cmd/chainsimulator/main.go

```diff
@@ -395,16 +395,27 @@ func loadMainConfig(filepath string) (config.Config, error) {
 	return cfg, err
 }
 
-func determineOverrideConfigFiles(ctx *cli.Context) []string {
+func determineOverrideConfigFiles(ctx *cli.Context, isSovereign bool) []string {
```

Comment:
This function will make sure override files are in order
1. default node override
2. default sovereign node override
3. other node override files

Reply:
Can you please add this comment in the code?